core/src/main/java/co/aikar/commands/BaseCommand.java

```diff
@@ -479,26 +730,62 @@ private void executeCommand(CommandOperationContext commandOperationContext,
         }
     }
 
+    /**
+     * Whether the command can be executed by the issuer.
+     * <p>
```

Comment:
Actually go ahead and mark this deprecated and @see Conditions. This API was how you could do generic conditions before ,and now we have a proper api for that.